### PR TITLE
chore: update base images

### PR DESCRIPTION
**Pull request by michael1011**
Nothing exciting. Just updating the alpine base image

The build should be working now thanks to: #365

### CODE REVIEW

**Comment by kilrau** [comment]
arm64 build failed: https://travis-ci.org/github/ExchangeUnion/xud-docker/jobs/667786708?utm_medium=notification&utm_source=github_status

File: .travis.yml [hunk 1]
```diff
@@ -16,5 +16,6 @@ jobs:
   include:
     - stage: Create and push manifest lists
       script: tools/push --manifest-list
-    - stage: Test common cases
+
+    - stage: Unit Tests
       script: tools/test
```
Comment (reliveyy) [inline .travis.yml]: This `tools/test` script is the entrypoint to run all the unit tests and integration tests together. We only have one simple integration test case now. Another integration test case and some unit tests are added by #366. So "Unit Tests" is not the best description of `tools/test`.

Comment (michael1011) [reply .travis.yml:21]: Makes sense. I will undo my changes bug the naming of the Travis build stages is a little confusing right now:
image

Comment (reliveyy) [reply .travis.yml]: Yep. I knew the first stage is **"test"** which is not right. But I cannot make multi-archs work in "include" stage list. And I don't investigate much about how to change the default name "test". So I just leave it. We will resolve this confusing name in the future.

Comment (michael1011) [reply .travis.yml]: Alright no problem. I reversed my changes to the Travis build file. Ready to merge?

Comment (reliveyy) [reply .travis.yml]: Of cause!

**Pull request merged**